beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -111,6 +111,12 @@ pub static BLOCK_PROCESSING_POST_EXEC_PROCESSING: LazyLock<Result<Histogram>> =
             linear_buckets(5e-3, 5e-3, 10),
         )
     });
+pub static BLOCK_PROCESSING_DATA_COLUMNS_WAIT: LazyLock<Result<Histogram>> = LazyLock::new(|| {
+    try_create_histogram(
+        "beacon_block_processing_data_columns_wait_seconds",
+        "Time spent waiting for data columns to be computed before starting database write",
```

Comment:
Should add buckets?

Reply:
Addressed in 0a9d5a0f61b4114dff93fe88c46ca4b97dde75f2